admin/public/index.html

```diff
@@ -20,6 +20,7 @@
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
     <title>ClearGDPR - Management</title>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/bulma/0.7.1/css/bulma.min.css">
```

Comment:
Did @juanmnl also use bulma? Do we want it?

Reply:
I used Bulma since it's the boilerplate used in CT, but added it to avoid focusing on common components styling. Probably, the best way is to use it with Sass and remove not used dependencies like rebooting or grids.

Reply:
I think we can remove this, juans style is standalone.